### ISSUE TITLE
Send an email when a user's answer is received

### ISSUE THREAD

**Issue by maximusunc** [RobokopU24/qgraph#49]
We had this functionality on old Robokop. We currently have a signed in user's email, so the Express server could potentially send the user an email when it receives a response back from an ARA.

**Comment by cbizon** [comment]
FWIW, I don't find these emails very useful - I wouldnt' mind if it was gone, or configurable to turn off.

**Comment by maximusunc** [comment]
@patrickkwang @kennethmorton do you guys have any input?

**Comment by patrickkwang** [comment]
I agree with @cbizon. If this was an option you could turn off, I don't think anyone would complain, but I consider it a very low-priority feature for development.

**Comment by maximusunc** [comment]
I'm envisioning a little checkbox near the submit button that says something like "Send me an email when I get answers back" which will default be unchecked.
